devicetypes/smartthings/zwave-motion-light-sensor.src/zwave-motion-light-sensor.groovy

```diff
@@ -184,21 +179,21 @@ def zwaveEvent(physicalgraph.zwave.commands.sensormultilevelv5.SensorMultilevelR
 		default:
 			map.descriptionText = cmd.toString()
 	}
-	results += createEvent(map)
+	results << createEvent(map)
 	return results
 }
 
 def zwaveEvent(physicalgraph.zwave.commands.wakeupv2.WakeUpNotification cmd) {
 	def results = []
-	results += createEvent(descriptionText: "$device.displayName woke up", isStateChange: false)
-	results += response(zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 3, scale: 1).format())
+	results << createEvent(descriptionText: "$device.displayName woke up", isStateChange: false)
+	results << response(zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 3, scale: 1).format())
 	if (!state.lastbatt || (now() - state.lastbatt) >= 10 * 60 * 60 * 1000) {
-		results += response(["delay 1000",
+		results << response(["delay 1000",
```

Comment:
I believe for this `response` the result will be a list, so you will want to use `+=`.

Reply:
I've only reverted my changes, so it went back as it was before I've started editing.